config/src/server/mod.rs

```diff
@@ -57,6 +60,9 @@ pub struct Network {
     /// Server port
     #[serde(default = "defaults::server_port")]
     pub port: u16,
+    /// Base public URL of server
+    #[serde(default = "defaults::base_url")]
```

Comment:
I'm thinking if that really should have a default value. Maybe let's change it to `Option<Url>`? And print some error if it was None when trying to get it on Google Login.

Reply:
Done, I've made it fallback to the server address and port, which will work for local testing.